packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -418,6 +420,50 @@ export function getResolver({
             }
           }
 
+          // Apply autogenerated slicing arguments
+          if (data.options.addSlicingPattern && Array.isArray(saneData)) {
+            // NOTE: Does not differentiate between autogenerated args and
+            // preexisting args
+            if ('first' in args && 'last' in args) {
```

Comment:
So, we are relying on a convention here, namely that the slicing arguments are named `first` or `last`. I think we can do more research on common names in our GitHub dataset. But even then, maybe the user needs to provide names of slicing arguments?